webpack/assets/javascripts/react_app/components/Layout/Layout.js

```diff
@@ -26,6 +27,7 @@ class Layout extends React.Component {
       isCollapsed,
       onCollapse,
     } = this.props;
+    checkBrowserSupport();
```

Comment:
I'm not sure if `Layout` is the best place for checking browser support, it doesn't really related to the layout (verical nav / top bar), so I concern it will become a place for global stuff.
Another option is to add this check into `ReactApp`, which is the root component.